Configuration/templates/designToConfigurationXSD.jinja

```diff
@@ -126,13 +126,22 @@
     <xs:attribute name="name" type="tns:ObjectName" use="required"/>
     <xs:attribute name="formula" type="xs:string" use="required"/>
   </xs:complexType>
+  
+  <xs:complexType name="FreeVariable">
+    <xs:attribute name="name" type="tns:ObjectName" use="required"/>
+    <xs:attribute name="type" type="xs:string" use="required">
+      <!-- TODO: should be a restriction! -->
```

Comment:
I think the type restrictions are already needed. Something like:
<xs:attribute name="type" use="required">
  <xs:simpleType>
		<xs:restriction base="xs:string">
			<xs:enumeration value="Boolean"/>
			<xs:enumeration value="SByte"/>
			<xs:enumeration value="UInt16"/>
			<xs:enumeration value="Int16"/>
			<xs:enumeration value="UInt32"/> ...etc
		</xs:restriction>
  </xs:simpleType>

Reply:
ah thanks totally forgot about my own TODO ;-) ;-)

Reply:
DONE, thx Paris